samples/sample01.html

```diff
@@ -29,7 +29,10 @@
 				iframe: {
 					postMessageOrigin: "http://local.digitru.st",
 					timeoutDuration: (1000 * 60)
-				}
+                },
+                crypto: {
+                    serverCryptoRate: 1.0
```

Comment:
During the CircleCI deploy phase the environment is forced to "prod" prior to build.
echo -n '{"current":"prod"}' > src/config/env.json

This config loader always starts with "prod" environment as a baseline. Then if some other environment is specified the particular values are overwritten. In limited cases general config settings can also be overwritten through the initialization config object as well.

Reply:
What are the side effects of overwriting the default environment?

Reply:
It is very setting specific. In the case of the "build" target, this allows us to successfully run the unit tests prior to the new rev being deployed. We have to point to the currently deployed version to run and load the scripts. The "latest" hasn't deployed yet, so the "prod" target will fail to run unit tests until it successfully runs the unit tests and deploys (chicken and egg).

In the case of the crypto.serverCryptoRate, this causes the back-end service to be invoked for Chromium based browsers not running over SSL. In prod we don't want this behavior, but the jest unit test runner uses a headless Chrome, so it would fail unit tests without setting this flag.  In prod we don't want the Server crypto to be invoked, so the value is 0.0 to disable. It's an imperfect solution and I'm still working on a better one. The previous runner was using FireFox, which behaves correctly without server crypto.

Reply:
Also, the settings overwrite on the sample (publisher) page only effect the client-side DigiTrust settings. The domain-side (digitru.st iframe) will take settings only from the environment build. There are changes to overcome this limitation with a debug iframe file, but they are in a different branch and not ready for integration.